ansible/configs/ent-demo-ate-base/default_vars.yml

```diff
@@ -60,7 +61,7 @@ deploy_local_ssh_config_location: "{{ output_dir }}/"
 
 # agd_set_env_authorized_key: true
 
-set_env_authorized_key: true  #"{{ agd_set_env_authorized_key }}" # Role should default this
+set_env_authorized_key: true
 env_authorized_key: "{{ guid }}key"
```

Comment:
Those are deprecated, see https://redhat-cop.github.io/agnosticd/#_deprecated_roles_and_variables

Reply:
Can I have a pass for now and will fix by end of first week in April @fridim